babel/dates.py

```diff
@@ -395,8 +402,12 @@ def get_timezone_gmt(datetime=None, width='long', locale=LC_TIME):
 
     :param datetime: the ``datetime`` object; if `None`, the current date and
                      time in UTC is used
-    :param width: either "long" or "short"
+    :param width: either "long" or "short" or "iso8601"
     :param locale: the `Locale` object, or a locale string
+    :param return_z: True or False; Function returns indicator "Z"
+                     when local time offset is 0
+    :param x_format: True or False; Function returns The ISO8601 basic format
```

Comment:
The `z` flag is fine, but I think this flag needs to be rethought, since it looks like it only affects one situation anyway.

Maybe make this another `width`? `iso8601_short`, for instance?

So `iso8601` would always return the +HH:MM format, and `iso8601_short` would return either `+HH` or `+HH:MM`?